### PR TITLE
AI Proxy Persistence

### PR DESCRIPTION
**Pull request by Iamdavidonuh**

### CODE REVIEW

File: ahnlich/ai/src/tests/aiproxy_test.rs [hunk 1]
```diff
@@ -446,3 +457,129 @@ async fn test_ai_proxy_fails_db_server_unavailable() {
...
+    let message = AIServerQuery::from_queries(&[
+        AIQuery::CreateStore {
+            r#type: AIStoreType::RawString,
+            store: store_name.clone(),
+            model: AIModel::Llama3,
+            predicates: HashSet::from_iter([]),
+            non_linear_indices: HashSet::new(),
+        },
+        AIQuery::ListStores,
+        AIQuery::DestoryDatabase,
```
Comment (deven96) [inline ahnlich/ai/src/tests/aiproxy_test.rs:566]: More like destroy store or something

**Review by deven96 — CHANGES_REQUESTED** [review]

File: ahnlich/ai/src/tests/aiproxy_test.rs (hunk 1 above)
Comment (Iamdavidonuh) [reply ahnlich/ai/src/tests/aiproxy_test.rs:566]: destroy stores ?
It's a db wide action

Comment (deven96) [reply ahnlich/ai/src/tests/aiproxy_test.rs:566]: Yes but we can't refer to the ai store as a database.... might also falsely give the impression that we are destroying the actual database i.e the vector db

**Review by deven96 — DISMISSED** [review]

**Review by deven96 — DISMISSED** [review]

File: type_specs/query/ai_query.json [hunk 2]
```diff
@@ -174,6 +174,9 @@
         "ListStores": "UNIT"
       },
       "10": {
+        "PurgeStores": "UNIT"
```
Comment (deven96) [inline type_specs/query/ai_query.json:177]: Could you modify the Doc [here](https://github.com/deven96/ahnlich/blob/main/docs/ai.md) to reflect all ai queries accurately

**Review by deven96 — APPROVED** [review]

**Pull request merged**